src/test/java/data/CameraShotTest.java

```diff
@@ -32,5 +32,17 @@ public void setNameTest() {
         cameraShot.setName("testname");
         assertEquals("testname", cameraShot.getName());
     }
+    
+    @Test
+    public void getDescriptionTest() {
+        assertEquals("some-description", cameraShot.getDescription());
+        
+    }
+    
+    @Test
+    public void setDescriptionTest() {
+        cameraShot.setDescription("a");
+        assertEquals("a", cameraShot.getDescription());
```

Comment:
Are tests necessary for lombok generated getters/setters included in the coverage?